packages/core/src/external/carequality/ihe-gateway-v2/saml/saml-client.ts

```diff
@@ -69,10 +69,12 @@ export async function sendSignedXml({
   signedXml,
   url,
   samlCertsAndKeys,
+  isDq,
```

Comment:
I only want to increase number of retries for dq, but xcpd and dq use the same client function

Reply:
Could you use the Url? istead of adding a param

Reply:
I thought this was the url to the api all good